kolibri/core/auth/management/utils.py

```diff
@@ -55,3 +69,96 @@ def get_facility(facility_id=None, noninteractive=False):
                 facility = _interactive_facility_selection()
 
     return facility
+
+
+def get_dataset_id(baseurl, identifier=None):
```

Comment:
Identifier can now either be a `facility_id` or a `dataset_id` because older versions of kolibri only accepted `dataset_id`.